### PR TITLE
#343-performed-database-migration

### PR DESCRIPTION
**Pull request by rishavsarma5**
Screenshot (340)

https://docs.google.com/document/d/1y1hMYbGLkjrgl3DwfgE6dtVjzpqiDjN-zltIpPz2VEU/edit?usp=sharing

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,8 +2,6 @@
 
 .eslintcache
 
-.env
```
Comment (jamescd18) [inline .gitignore:5]: No. What does the .gitignore do?

File: .env [hunk 2]
```diff
@@ -0,0 +1 @@
+DATABASE_URL="postgresql://postgres:Xboxnam101@localhost:5432/nerpm?schema=public"
```
Comment (jamescd18) [inline .env:1]: This should not be in the codebase. You have exposed your local database password to the public.

File: package.json [hunk 3]
```diff
@@ -189,7 +189,7 @@
     "netlify-lambda": "^2.0.3",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.2.1",
-    "prisma": "~2.19.0",
+    "prisma": "^2.19.0",
```
Comment (jamescd18) [inline package.json:192]: You should not be changing versions without a very strong reason

File: src/backend/prisma/migrations/20211221014206_test/migration.sql [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ALTER TABLE "WBS_Element" DROP COLUMN "projectLeadId",
+ADD COLUMN     "projectLeadId" INTEGER[];
+
+-- CreateTable
+CREATE TABLE "_projectLead" (
+    "A" INTEGER NOT NULL,
+    "B" INTEGER NOT NULL
+);
+
+-- CreateIndex
```
Comment (jamescd18) [inline src/backend/prisma/migrations/20211221014206_test/migration.sql:17]: We don't want any changes to the migrations to get added to the codebase.

**Review by jamescd18 — COMMENTED** [review]
This needs some more explanation please. I see you've made changes to the codebase and linked a google doc. 

If you want to use a Google doc to iterate on the instructions before putting them into the proper codebase docs files, I'm okay with that. That said, at a bare minimum you're going to need to give me edit access so I can leave feedback, and you also should place the file in the correct NER Google Drive folder. That said, I don't recommend using a Google doc since the docs properly live in the codebase, and GitHub provides a great line-by-line feedback tool via pull requests. 

As for all of the changes that I see here in the codebase, they are all incorrect. I've specified why in inline comments, but I would strongly recommend doing a little research into what the gitignore does and why we don't put the .env in the codebase. If you aren't able to find answers, let me know and I can explain.

File: .gitignore (hunk 1 above)
Comment (jinhyunh) [reply .gitignore:5]: It chooses which files to ignore committing the project onto the repository

**Pull request closed without merging**